test/plenario_web/controllers/api/aot_controller_test.exs

```diff
@@ -165,6 +165,48 @@ defmodule PlenarioWeb.Api.AotControllerTest do
       assert meta["counts"]["total_records"] == @total_records
     end
 
+    test "invalid bbox with missing coordinate", %{conn: conn} do
+      min_lat = 41.95
+      max_lat = 41.97
+      min_lon = -87.65
+      max_lon = -87.67
+      bbox = %Geo.Polygon{
+        srid: 4326,
+        coordinates: [[
+          {min_lat, max_lon},
+          {min_lat, min_lon},
+          {max_lat, min_lon},
+          {max_lat, max_lon}
+          # missing closing coordinate
+        ]]
+      } |> Geo.JSON.encode()
+
+      assert_raise Ecto.SubQueryError, fn ->
+        get(conn, "/api/v2/aot?bbox=#{bbox}")
+      end
+    end
+
+    test "invalid bbox with unacceptable coordinates", %{conn: conn} do
+      min_lat = -10000000
+      max_lat = "bounding"
+      min_lon = "box"
+      max_lon = -10000000
+      bbox = %Geo.Polygon{
+        srid: 4326,
+        coordinates: [[
+          {min_lat, max_lon},
+          {min_lat, min_lon},
+          {max_lat, min_lon},
+          {max_lat, max_lon}
+          # missing closing coordinate
```

Comment:
I think this would be a better test if we just used

```elixir
%Geo.Polygon{
  srid: 4326,
  coordinates: [[
    {max_lon, max_lat},
    {max_lon, min_lat},
    {min_lon, min_lat},
    {min_lon, max_lat},
    {max_lon, max_lat}
  ]]
}
```

to induce the database error.

Reply:
Ah oops